resim/ros2/converter_plugin.cc

```diff
@@ -15,6 +16,49 @@
 
 namespace resim::ros2 {
 
+namespace {
+
+// Simple helper to make sure we initialize the buffer with nullptr. T
+// can be rmw_serialized_message_t or rcutils_uint8_array_t.
```

Comment:
Ah! It is technically (https://docs.ros2.org/crystal/api/rcl/types_8h.html#aacac4adc45d07815b73188eafd77d705) but better to use rcl_serialized_message_t since that's the alias used in this file. 👍